dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Shared/RestError.cshtml

```diff
@@ -29,10 +29,17 @@
     if (commonError != null)
     {
         ErrorModel deserializeObject = JsonConvert.DeserializeObject<ErrorModel>(commonError.Response.Content);
```

Comment:
I am not sure what do you mean by this comment, could you be more specific?

---
In reply to: [117381816](https://github.com/Microsoft/healthvault-samples/pull/10#discussion_r117381816) [](ancestors = 117381816)